pkg/gci/gci.go

```diff
@@ -141,3 +145,25 @@ func LoadFormatGoFile(file io.FileObj, cfg GciConfiguration) (unmodifiedFile, fo
 	}
 	return unmodifiedFile, formattedFile, nil
 }
+
+// isGenerated reports whether the source file is generated code.
+// Using a bit laxer rules than https://golang.org/s/generatedcode to
+// match more generated code.
+// Taken from https://github.com/golangci/golangci-lint.
+func isGeneratedFileByComment(doc string) bool {
+	const (
+		genCodeGenerated = "code generated"
+		genDoNotEdit     = "do not edit"
+		genAutoFile      = "autogenerated file" // easyjson
+	)
+
+	markers := []string{genCodeGenerated, genDoNotEdit, genAutoFile}
+	doc = strings.ToLower(doc)
```

Comment:
I would keep case sensitive.

Reply:
This is taken directly from golangci-lint and I think it's fine. Not every code generation tool does it the same way.